docker-compose-production.yml

```diff
@@ -4,6 +4,11 @@
 version: '3'
 
 services:
+  app: &app
+    build:
+      args:
+        FLUENTD_URL: ${FLUENTD_URL}
```

Comment:
its not clear to me that this build arg gets used anywhere in this pr.  is it needed?

Reply:
Initially FLUENTD_URL was in a config file and the build failed without the env.
https://github.com/actindi/act-fluent-logger-rails#create-configfluent-loggeryml

I moved the configuration to ``.env`` file and removed this entry from the yaml file.